search_service/models/table.py

```diff
@@ -18,6 +19,7 @@ class Table(Base):
     key: str
     display_name: Optional[str] = None
     tags: List[str]
```

Comment:
we should do the same for tags, badges

Reply:
> we should do the same for tags, badges

Same comment. We can use the one in `amundsen-common`
https://github.com/lyft/amundsencommon/blob/b52eae378eaec39cc538e969391513c4ccad59a6/amundsen_common/models/table.py#L22

Reply:
i ended up going with the model in amundsen-common.